.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+
+workflows:
+    build:
+      jobs:
+        - test
+
+jobs:
+  test:
+    docker:
+      - image: cimg/ruby:3.1.2
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install
+      - run:
+          name: Run tests
+          command: rspec
+      - run:
+          name: Run rubocop
+          command: rubocop
+      - run:
+          name: Run fasterer
+          command: fasterer
```

Comment:
It's better to set linters and testing in separate commands (also u need to set default command for installing dependencies). Also u need to create separate jobs for them and run tests only if linters have succeeded.